rules-reviewed/eap7/eap6/eap6-xml.windup.xml

```diff
@@ -57,12 +57,9 @@
         </rule>
         <rule id="eap6-xml-03000">
             <when>
-                <xmlfile matches="/webservices/webservice-description/jaxrpc-mapping-file | /jee:webservices/jee:webservice-description/jee:jaxrpc-mapping-file | /jcp:webservices/jcp:webservice-description/jcp:jaxrpc-mapping-file | /j2ee:webservices/j2ee:webservice-description/j2ee:jaxrpc-mapping-file">
-                    <namespace prefix="j2ee" uri="http://java.sun.com/xml/ns/j2ee"/>
-                    <namespace prefix="jcp" uri="http://xmlns.jcp.org/xml/ns/javaee"/>
-                    <namespace prefix="jee" uri="http://java.sun.com/xml/ns/javaee"/>
-
-                </xmlfile>
+                <xmlfile matches="/*[contains(local-name(), 'webservices')]/*[contains(local-name(), 'webservice-description')]/*[contains(local-name(), 'jaxrpc-mapping-file')]"/>
+                <xmlfile matches="/*[contains(local-name(), 'ejb-jar')]/*[contains(local-name(), 'enterprise-beans')]/*[contains(local-name(), 'session')]/*[contains(local-name(), 'service-ref')]/*[contains(local-name(), 'jaxrpc-mapping-file')]"/>
+                <xmlfile matches="/*[contains(local-name(), 'java-wsdl-mapping')]"/>
```

Comment:
I'd prefer keeping the namespaces, it's more safe. (See also comments in https://issues.jboss.org/browse/JBEAP-3141)